src/breed/schema/breed.schema.ts

```diff
@@ -0,0 +1,15 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { HydratedDocument } from 'mongoose';
+
+export type BreedDocument = HydratedDocument<breed>;
+
+@Schema()
+export class breed {
+  @Prop({ required: true })
+  name: string;
+
+  @Prop({ required: true, minlength: 6 })
```

Comment:
remove the minlength property because there are animals whose names are shorter than 6 characters such as cats

Reply:
Fixed